website/src/components/RsvWastewaterInfo.astro

```diff
@@ -0,0 +1,29 @@
+---
+interface Props {
+    reference: string;
+}
+
+const { reference } = Astro.props;
+
+/* eslint-disable @typescript-eslint/naming-convention */
+const additionalInfo = {
+    'RSV-A': {
+        referenceSequence: 'EPI_ISL_412866 (hRSV/A/England/397/2017)',
+    },
+    'RSV-B': {
+        referenceSequence: 'EPI_ISL_1653999 (hRSV/B/Australia/VIC-RCH056/2019)',
+    },
+};
+/* eslint-enable @typescript-eslint/naming-convention */
+
+if (!(reference in additionalInfo)) {
+    return null;
+}
```

Comment:
Nitpick: We know the possible values of `reference` for RSV, i.e. we could get rid of this check by giving `reference` the type `'RSV-A' | 'RSV-B'`.

Reply:
Let me know if this still looks good :-)